com.minres.coredsl.tests/src/com/minres/coredsl/tests/CoreDslInterpreterTest.xtend

```diff
@@ -35,9 +36,9 @@ class CoreDslInterpreterTest {
                 int a = 42;
                 int b = a + 5;
                 int XLEN = a + b;
-                [[is_pc]] int PC ;
-                int Xreg[XLEN];
-                float Freg[a];
+                register [[is_pc]] int PC ;
```

Comment:
[In the spec](https://github.com/Minres/CoreDSL/wiki/Structure-and-concepts#registers), attributes come after the ID.

Reply:
The grammar allows them in both places. I assume attributes before the type specifier apply to the whole declaration while those after each declarator name are specific to the indivitual declarators?

Reply:
Hmm, you mean like:
```
register [[attrib_all]] int a [[attrib_a]], b [[attrib_b]];
```
Idk, this looks more like a leftover from an earlier version to me. @eyck WDYT? I'd vote to only allow attributes after the ID (relating to that specific declaration).

Reply:
Removed the first attribute location from the grammar.